package.json

```diff
@@ -9,10 +9,14 @@
   "repository": "https://github.com/brafdlog/caspion",
   "homepage": "https://github.com/brafdlog/caspion#README.md",
   "scripts": {
-    "serve": "vue-cli-service electron:serve",
+    "ui": "cd ui-react && yarn start",
+    "serve": "concurrently --kill-others-on-fail \"yarn serve-electron\"  \"yarn ui\"",
+    "serve-electron": "vue-cli-service electron:serve",
+    "serve-electron:debug": "set DEBUG=israeli-bank-scrapers:* & vue-cli-service electron:serve",
     "serve:docker": "yarn serve --no-sandbox",
     "build": "vue-cli-service electron:build",
-    "postinstall": "electron-builder install-app-deps",
+    "prebuild": "cd ui-react && yarn && yarn build",
+    "postinstall": "electron-builder install-app-deps && cd ui-react && yarn && cd ..",
```

Comment:
This is on purpose. It creates a separation of concerns between the UI code and electron code.
As we discussed a few times, this could easily become a cli, or use a different UI. Separating the package.json files allows us to have separate dependencies and will make this project more modular.

Reply:
Sure, I just think each package.json should be in a subfolder for its project, and maybe use `yarn workspaces` or something. But not now, we will consider it next time, because there is a pain in the separation.